packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte

```diff
@@ -0,0 +1,193 @@
+				control: 'text'
+			},
+			intervalTop: {
+				control: 'text'
+			},
+			max: {
+				control: 'text'
+			},
+			confidenceInterval: {
+				control: 'text'
+			},
+			emptySet: {
+				control: 'select',
+				options: ['pass', 'warn', 'error']
+			},
+			emptyMessage: {
+				control: 'text'
+			},
+			color: {
+				control: 'text'
+			},
+			yFmt: {
+				control: 'text'
+			},
+			xFmt: {
+				control: 'text'
+			},
+			seriesColors: {
+				control: 'object'
+			},
+			swapXY: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			xAxisTitle: {
+				control: 'text'
+			},
+			yAxisTitle: {
+				control: 'text'
+			},
+			xGridlines: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			yGridlines: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			xBaseline: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			yBaseline: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			xTickMarks: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			yTickMarks: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			yMin: {
+				control: 'number'
+			},
+			yMax: {
+				control: 'number'
+			},
+			showAllAxisLabels: {
+				control: 'boolean',
+				options: [true, false]
+			},
+			title: {
+				control: 'text'
+			},
+			subtitle: {
+				control: 'text'
+			},
+			chartAreaHeight: {
+				control: 'number'
+			},
+			renderer: {
+				control: 'select',
+				options: ['canvas', 'svg']
+			},
+			eChartsOptions: {
+				control: 'object'
+			},
+			seriesOptions: {
+				control: 'object'
+			},
+			printEchartsConfig: {
+				control: 'boolean',
+				options: ['true', 'false']
+			}
+		}
+	};
+</script>
+
+<script>
+	import { Story } from '@storybook/addon-svelte-csf';
+	import BoxPlot from './BoxPlot.svelte';
+	import { Query } from '@evidence-dev/sdk/usql';
+	import { query } from '@evidence-dev/universal-sql/client-duckdb';
+	import { writable } from 'svelte/store';
+	import { setContext } from 'svelte';
+	import { INPUTS_CONTEXT_KEY } from '@evidence-dev/component-utilities/globalContexts';
+	const inputStore = writable({});
+	setContext(INPUTS_CONTEXT_KEY, inputStore);
+	const flightData = Query.create(
+		`
+SELECT 
+    airline, 
+    MAX(fare) AS max_fare, 
+    MIN(COALESCE(fare, 0)) AS min_fare, 
+    MEDIAN(fare) AS median_fare,
+	MAX(fare)-1000 AS intervalTop_fare,
+	MIN(fare)+1000 AS intervalBottom_fare,
+    CASE 
+        WHEN airline = 'Qatar Airways' THEN 'red' 
+        WHEN airline = 'AirAsia' THEN 'blue' 
+        ELSE 'gray' 
+    END AS color
+FROM flights
+WHERE airline IN ('Qatar Airways', 'AirAsia')
+GROUP BY airline, color
+limit 50`,
+		query
+	);
+</script>
+
+<Story
+	name="Base"
+	args={{
+		name: 'airline',
+		intervalBottom: 'min_fare',
+		midpoint: 'median_fare',
+		intervalTop: 'max_fare',
+		yFmt: 'usd0'
+	}}
+	let:args
+>
+	<BoxPlot {...args} data={flightData} />
+</Story>
+
+<Story
+	name="swapXY=true colors"
+	args={{
+		color: 'color',
```

Comment:
`color` isn't a valid color, so this does nothing

Reply:
color refers to the name of column type defined in the sql query.
![image](https://github.com/evidence-dev/evidence/assets/69778837/3877f71f-bbff-49bc-bac5-bdea07a9adb5)
![image](https://github.com/evidence-dev/evidence/assets/69778837/e8da62bf-e6c0-4a95-9801-512e5dfd5f04)

	
Sorry poor naming convention